### PR TITLE
Add form to the fieldset example

### PR DESCRIPTION
**Pull request by Aronja**
- this example seemed to confuse students, as the fieldset was not
in the form

### CODE REVIEW

File: html-css/week-03/lesson.md [hunk 1]
```diff
@@ -228,6 +231,7 @@ Checkboxes that are related to each other can be grouped together in a `<fieldse
     English
   </label>
 </fieldset>
+</form>
```
Comment (NateWr) [inline html-css/week-03/lesson.md:234]: This looks good, thanks. Can you up the indentation for the nested tags in both examples?

File: html-css/week-03/lesson.md [hunk 2]
```diff
@@ -193,14 +193,15 @@ A checkbox allows the user to say yes or no to an option. With a checkbox, the u
 
 ---
 
-Checkboxes that are related to each other can be grouped together in a `<fieldset>`. A group of fields is described by a `<legend>`.
+Checkboxes that are related to each other can be grouped together in a `<fieldset>`. A group of fields is described by a `<legend>`. As you can see a fieldset (similar to a field) needs to be inside a form.
```
Comment (NateWr) [inline html-css/week-03/lesson.md:196]: I'd like to leave this out. The code example demonstrates this in a clearer way than text anyway, and in my view it helps if the explanation bits are as lean and focused as possible.

**Review by NateWr — COMMENTED** [review]
Thanks! There are two more areas where the fieldsets are used:

- https://github.com/CodeYourFuture/syllabus/blob/scotland/html-css/week-03/lesson.md#fieldsets
- https://github.com/CodeYourFuture/syllabus/blob/scotland/html-css/week-03/lesson.md#checked

Can you update those too?

File: html-css/week-03/lesson.md (hunk 1 above)
Comment (Aronja) [reply html-css/week-03/lesson.md:234]: yeah sure!

File: html-css/week-03/lesson.md (hunk 2 above)
Comment (Aronja) [reply html-css/week-03/lesson.md:196]: I would leave it in... the students are still new to coding so for some of them just seeing it in code might not be that clear. I think having a short extra sentence in there explaining it in text, takes everyone along the journey also they one who aren't that familiar with spotting differences in code

**Comment by Aronja** [comment]
I only updated the first fieldset example deliberately, as I thought updating all of them might be convoluted and I though we wanted to keep the examples as short as possible. That's why I added the one line of text to make it clear to all students in case they aren't sure

File: html-css/week-03/lesson.md (hunk 2 above)
Comment (NateWr) [reply html-css/week-03/lesson.md:196]: I guess this makes me more concerned about inconsistency. All the elements we cover in this lesson need to be inside of a form tag, so I wonder if it's confusing to say it for fieldsets but not other fields.

Maybe we need to cover this more explicitly in the lesson. I'm thinking maybe once at the end of The HTML Form section, once at the end of Fields, once at the end of Labels and once at the end of Buttons?

**Pull request closed without merging**